src/module/item/data-model-ability.js

```diff
@@ -21,15 +21,21 @@ export default class OseDataModelAbility extends foundry.abstract.DataModel {
   get #rollTag() {
     if (!this.roll) return null;
 
+    const formulaData = {
```

Comment:
I am irrationally bothered by this code being duplicated. Thoughts on extracting it to a helper method?

I *really* wish extending a base data model class didn't seem to break data models.

Reply:
I could consolidate them by adding a method to the dice helper, is this what you are thinking? Would need to import the helper obviously.
```
    return {
      label: `${game.i18n.localize("OSE.items.Roll")} ${OseDice.rollFormula({
        actor: this.parent.actor,
        data: this._source,
      })
        }${rollTarget}`,
```

Reply:
I took a stab at adding a new helper in typescript for this in https://github.com/vttred/ose/pull/499/commits/7ecdf48d30fa72471dcb518753ab6e49fa1c276c